### ISSUE TITLE
Getting wall info from URDF instead of hardcoding

### ISSUE THREAD

**Issue by Erfi** [shamilmamedov/flexible_arm#41]
**Description**:
Currently we are putting hard-coded values here and there for the env, controller and the safety filter. This is quite error prone. Instead we should be able to take this info from else where and store it in the env and other modules can take this info from the env. 

**Implementation Suggestion**:
If it is possible to infer the vectors that describe the wall from URDF then we can build a model from the URDF inside the env and infer all the wall info from there. Otherwise perhaps storing them in a file and loading it from there (or later from a config file like hydra?)

**Comment by RudolfReiter** [comment]
I agree, this is very important. Right now I don't even dare to change the wall parameters in the MPC. I would suggest it should be stored in the URDF?
